docs/UserGuide.md

```diff
@@ -15,30 +22,35 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+1. Ensure you have **Java 17 or above** installed in your Computer.
+   *  How To Install Java ([Windows](https://se-education.org/guides/tutorials/javaInstallationWindows.html) | [MacOS](https://se-education.org/guides/tutorials/javaInstallationMac.html) | [Linux](https://se-education.org/guides/tutorials/javaInstallationLinux.html))
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14b-1/tp/releases).
+   * Under the **Assets** of the latest version of SocialBook, you should find the latest downloadable `socialbook.jar` file
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy this file into the _folder_ you want to use for SocialBook.
+   * Ensure that this _folder_ is **empty**.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Open a command terminal, `cd` into the folder you put the `.jar` file in. After which enter `java -jar socialbook.jar` in the command terminal to run the application.
```

Comment:
Can look into quoting resource to find current working directory for digitally illiterate user